workspaces/liferay-sample-workspace/package.json

```diff
@@ -10,5 +10,24 @@
 		"prettier": "3.2.5",
 		"typescript": "4.4.2"
 	},
-	"private": true
-}
+	"private": true,
+	"workspaces": {
+		"packages": [
+			"client-extensions/liferay-sample-fds-cell-renderer",
+			"client-extensions/liferay-sample-theme-css-2",
+			"client-extensions/liferay-sample-theme-css-3",
+			"client-extensions/liferay-sample-etc-node",
+			"client-extensions/liferay-sample-etc-frontend",
+			"client-extensions/liferay-sample-js-import-maps-entry",
+			"client-extensions/liferay-sample-editor-config-contributor",
+			"client-extensions/liferay-sample-theme-spritemap-2",
+			"client-extensions/liferay-sample-fds-filter",
+			"client-extensions/liferay-sample-theme-css-1",
+			"client-extensions/liferay-sample-commerce-checkout-step",
+			"client-extensions/liferay-sample-custom-element-5",
+			"client-extensions/liferay-sample-custom-element-2",
+			"client-extensions/liferay-sample-custom-element-3",
+			"client-extensions/liferay-sample-custom-element-4"
+		]
+	}
+}
```

Comment:
This thing always auto-generates for me when I build workspace CX samples, along with new `.yarnrc` file. I always discard it. I don't know if there is a reason that we don't commit those changes.,